docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.rst

```diff
@@ -0,0 +1,108 @@
+        "tasks.max":"NR_TASKS",
+        "key.converter": "io.confluent.connect.avro.AvroConverter",
+        "key.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "key.converter.basic.auth.credentials.source": "USER_INFO",
+        "key.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "value.converter.basic.auth.credentials.source": "USER_INFO",
+        "value.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``PG_HOST``, ``PG_PORT``, ``PG_DATABASE_NAME``, ``SSL_MODE``, ``PG_USER``, ``PG_PASSWORD``, ``PG_TABLES``, ``PG_PUBLICATION_NAME`` and ``PG_SLOT_NAME``: source database parameters collected in the :ref:`prerequisite <connect_debezium_pg_source_prereq>` phase. 
+* ``database.server.name``: the logical name of the database, dictates the prefix that will be used for Apache Kafka topic names. The resulting topic name will be the concatenation of the ``database.server.name`` and the table name.
+* ``tasks.max``: maximum number of tasks to execute in parallel. By default is 1, the connector can use at max 1 task for each source table defined.
+* ``plugin.name``: defines the `PostgreSQL output plugin <https://debezium.io/documentation/reference/connectors/postgresql.html>`_ to convert changes in the database into events in Apache Kafka.
+* ``key.converter`` and ``value.converter``:  defines the messages data format in the Apache Kafka topic. The ``io.confluent.connect.avro.AvroConverter`` converter pushes messages in Avro format. To store the messages schema we use Aiven's `Karapace schema registry <https://github.com/aiven/karapace>`_ pointed by the ``schema.registry.url`` parameter and related credentials.
+
+.. Note::
+
+    The ``key.converter`` and ``value.converter`` sections are only needed when pushing data in Avro format. If omitted the messages will be defined in JSON format.
+
+
+Create a Kafka Connect connector with Aiven CLI
+'''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, execute the following :ref:`Aiven CLI command <avn_service_connector_create>`, replacing the ``SERVICE_NAME`` with the name of the Aiven service where the connector needs to run:
+
+:: 
+
+    avn service connector create SERVICE_NAME @debezium_source_pg.json
+
+Check the connector status with the following command, replacing the ``SERVICE_NAME`` with the Aiven service and the ``CONNECTOR_NAME`` with the name of the connector defined before:
+
+::
+
+    avn service connector status SERVICE_NAME CONNECTOR_NAME
+
+Verify in the Apache Kafka target instance, the presence of the topic and the data
```

Comment:
I think this might be better if you flip the order, to something more like `Verify the presence of the topic and data in the Apache Kafka target instance`.

Reply:
flipped!